challenge.rb

```diff
@@ -67,6 +95,14 @@ def ac_monitor current_temp, ac_working, desired_temp
 # and print out the result of `fizz_buzz_calculator` for that number.
 # This question will be graded independently of question four.
 
-# Replace this comment with your code.
 
+# not done-
+def fizz_buzz min max
+  while min < max
+    return fizz_buzz_calculator(min)
```

Comment:
Should be `print` or `puts` in this one - using `return` will break us out of the loop, so we won't finish retrieving the numbers. It also doesn't strictly print...